docs/api-reference/sql-api.md

```diff
@@ -827,6 +823,9 @@ When getting query results, keep the following in mind:
 * `page` (optional)
     * Type: Int
     * Refine paginated results
+* `resultFormat` (optional)
+    * Type: String
+    * Get results in different formats like `object`,`objectLines`,`array`,`arrayLines`,`csv`. Default is `object`.
```

Comment:
Can we reword this like how it is done in `query-from-deep-storage.md`. That seems cleaner because it presents the options like an exhaustive list of the supported parameters
```suggestion
    * Defines the format in which the results are presented. The following options are supported `arrayLines`,`objectLines`,`array`,`object`, and `csv`. The default is `object`.
```